### ISSUE TITLE
ImageSegmenter

### ISSUE THREAD

**Issue by haifeng-jin** [keras-team/autokeras#914]
One option would be using the auto deep lab search space

**Comment by haifeng-jin** [comment]
Follow the API of TimeSeriesForecaster.
https://github.com/keras-team/autokeras/tree/master/autokeras/tasks

**Comment by ECEMACHINE** [comment]
ImageSegmenter.fit(x,y)

**Comment by haifeng-jin** [comment]
@ECEMACHINE will design the APIs.

**Comment by haifeng-jin** [comment]
@ECEMACHINE will work on implementing the head.

**Comment by lc0** [comment]
@haifeng-jin it looks like the linked PR was closed. Is this one still planned?

**Comment by haifeng-jin** [comment]
Yes, but we don't have extra developers to work on it for now. This feature will be continued after object detection and time series data.
